cmd/ipfs/dist/install.sh

```diff
@@ -6,7 +6,7 @@
 INSTALL_DIR=$(dirname $0)
 
 bin="$INSTALL_DIR/ipfs"
-binpaths="/usr/local/bin /usr/bin"
+binpaths="$HOME/.local/bin /usr/local/bin /usr/bin"
```

Comment:
can we put this at the end of the list so that it's backwards compatible? (i.e. if you run w/ sudo it will continue to put the bin in the same place)

Reply:
Also will this work if $HOME has a space in it?

Reply:
@guseggert thanks for the review, the new commit should fix the issues you've mentioned.